src/lib/holocene/icon/svg.svelte

```diff
@@ -1,21 +1,17 @@
 <script lang="ts">
-  export let width = 24;
-  export let height = 24;
-  export let viewBox = '0 0 24 24';
   export let title = '';
 </script>
 
 <svg
-  {width}
-  {height}
-  {viewBox}
+  width="16"
+  height="16"
+  viewBox="0 0 24 24"
   fill="none"
   xmlns="http://www.w3.org/2000/svg"
   class={$$props.class}
   role="img"
   aria-hidden={!title}
-  {...$$restProps}
```

Comment:
We'll need to keep this so we can pass in x and y props for lines and dots icons. I would just changed the default width and height props to 16 and and leave:

  {width}
  {height}
  {viewBox}
  {...$$restProps}